### ISSUE TITLE
SmartCity-Accessibility #115 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#101]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 59 lines...]
[INFO] 
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ SmartCity-Accessibility ---
[INFO] Deleting /var/lib/jenkins/workspace/SmartCity-Accessibility/target
[INFO] 
[INFO] --- jacoco-maven-plugin:0.7.8:prepare-agent (default-cli) @ SmartCity-Accessibility ---
[INFO] argLine set to -javaagent:/var/lib/jenkins/.m2/repository/org/jacoco/org.jacoco.agent/0.7.8/org.jacoco.agent-0.7.8-runtime.jar=destfile=/var/lib/jenkins/workspace/SmartCity-Accessibility/target/jacoco.exec
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ SmartCity-Accessibility ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.5.1:compile (default-compile) @ SmartCity-Accessibility ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 52 source files to /var/lib/jenkins/workspace/SmartCity-Accessibility/target/classes
[INFO] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java: Some input files use or override a deprecated API.
[INFO] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java: Recompile with -Xlint:deprecation for details.
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/gui/components/ButtonsPanel.java:[34,29] cannot find symbol
  symbol:   class LoginFrame
  location: class smartcity.accessibility.gui.components.ButtonsPanel
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/gui/components/ButtonsPanel.java:[36,29] cannot find symbol
  symbol:   class SignUpFrame
  location: class smartcity.accessibility.gui.components.ButtonsPanel
[INFO] 2 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 15.168 s
[INFO] Finished at: 2016-12-27T11:54:52+00:00
[INFO] Final Memory: 28M/157M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.5.1:compile (default-compile) on project SmartCity-Accessibility: Compilation failure: Compilation failure:
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/gui/components/ButtonsPanel.java:[34,29] cannot find symbol
[ERROR] symbol:   class LoginFrame
[ERROR] location: class smartcity.accessibility.gui.components.ButtonsPanel
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/gui/components/ButtonsPanel.java:[36,29] cannot find symbol
[ERROR] symbol:   class SignUpFrame
[ERROR] location: class smartcity.accessibility.gui.components.ButtonsPanel
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [Alexander Kaplan] b864977f1f203925cd275443d7d52e39a9de7160 - added login menu #34

[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/115/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
